### PR TITLE
Cogged load_blocks for WID = {4,8}, various vecls, initializer_list for generalized vectorclass

### PR DESCRIPTION
**Pull request by alhom**
Compiles, _not tested._

### CODE REVIEW

**Comment by ykempf** [comment]
I think we need all to be explicit, as we could want to run AGNER with VECL == 8 and WID == 8. And we might want WID == 8 && VECL == 16, etc.

**Comment by alhom** [comment]
Those are there, but defs for VEC16+_AGNER are not. I'll add the explicit VECL == x's to the elifs and the new Agners.

**Comment by alhom** [comment]
Ah, now something actually broke.

**Comment by alhom** [comment]
So yup, Agners were there (silmä käteen). Everything's explicit and folded to the same format thanks to it being supported for Agner.

File: vlasovsolver/cpu_acc_load_blocks.cpp [hunk 1]
```diff
@@ -44,252 +44,505 @@ void loadColumnBlockData(
...
+         values[i_pcolumnv_b(0, 7, block_k, n_blocks)] = Vec({data[7], data[71], data[135], data[199], data[263], data[327], data[391], data[455], data[15], data[79], data[143], data[207], data[271], data[335], data[399], data[463], data[23], data[87], data[151], data[215], data[279], data[343], data[407], data[471], data[31], data[95], data[159], data[223], data[287], data[351], data[415], data[479], data[39], data[103], data[167], data[231], data[295], data[359], data[423], data[487], data[47], data[111], data[175], data[239], data[303], data[367], data[431], data[495], data[55], data[119], data[183], data[247], data[311], data[375], data[439], data[503], data[63], data[127], data[191], data[255], data[319], data[383], data[447], data[511]});
+   #else // Fell through, never fall into this particular pit again
+   	std::cerr << "Undefined VECTORCLASS flag or implementation missing in loadColumnBlockData() before " << __FILE__ << ":" << __LINE__ << std::endl;
```
Comment (ykempf) [inline vlasovsolver/cpu_acc_load_blocks.cpp]: This line as an example: I don't like indentation that's not a multiple of 3 spaces...

**Comment by ykempf** [comment]
I was looking for the WID 8, you wrapped it like so, I guess it works too and probably makes the cog more manageable. So apart from the indentation it looks good to me.

**Comment by alhom** [comment]
Done.

File: vlasovsolver/cpu_acc_load_blocks.cpp [hunk 2]
```diff
@@ -289,7 +255,294 @@ for dimension in range(0, 2):
             data[i]=0;
          }
       }
-   }
+    }
+   #elif WID == 8
```
Comment (ykempf) [inline vlasovsolver/cpu_acc_load_blocks.cpp:259]: Ahem. But I'll merge so we can proceed...

**Pull request merged**